### ISSUE TITLE
Language strings are not translated

### ISSUE THREAD

**Issue by obuisard** [joomla-projects/soc23_joomla-converter#42]
Reported by user.
Probably due to refactoring

![image](https://github.com/user-attachments/assets/ef5f275b-169a-4f48-8e22-2fe50c686854)

**Comment by Kaushik1216** [comment]
yes , I think the current code in main branch not properly migrated
